db/migrate/20220316152547_reset_feature_dismiss.rb

```diff
@@ -0,0 +1,5 @@
+class ResetFeatureDismiss < ActiveRecord::Migration[6.1]
+  def change
+    Publisher.update_all(dismissed_new_features_page_at: nil)
```

Comment:
This has been something that's been bothering me for a while, but it feels like instead of resetting this we should have a thing somewhere that has the date the features page has last been changed 😕

Reply:
(and then check against that)

Reply:
Yeah probably.  Maybe one for the supportal?